data/map beyond patir.txt

```diff
@@ -29,18 +29,13 @@ system Athiri
 	hazard "Small Black Hole Gravity" 1
 	object
 		sprite "star/black hole corona core"
-		period 1
 	object
 		sprite "star/black hole corona"
-		period 1
 	object "Beyond Patir North Entrance"
 		distance 1510.2286
-		period 1
-		offset 35
```

Comment:
Perhaps leave this offset? I'm not fully sure what it's meant for but assume it's intentional, as with the other end of the wormhole below.